data-products/pyproject.toml

```diff
@@ -15,6 +15,9 @@ snowflake-connector-python = "<3.0.4"
 prefect-snowflake = "^0.26.1"
 prefect-gcp = {extras = ["bigquery"], version = "^0.4.2"}
 jinja2 = "^3.1.2"
+pandas = "^2.0.3"
+db-dtypes = "^1.1.1"
+sagemaker = "^2.171.0"
```

Comment:
I believe it took over 15 minutes to lock this file, on the latest version of Poetry. :grimacing: Some quick Google searches didn't tell me much.

Reply:
The specific error is:

> google.api_core.exceptions.PermissionDenied: 403 request failed: the user does not have 'bigquery.readsessions.create' permission for 'projects/pocket-prefect-nonprod'